pallets/delegation/src/benchmarking.rs

```diff
@@ -0,0 +1,287 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2021 BOTLabs GmbH
+
+// The KILT Blockchain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The KILT Blockchain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// If you feel like getting in touch with us, you can do so at info@botlabs.org
+
+use codec::Encode;
+use frame_benchmarking::{account, benchmarks, impl_benchmark_test_suite};
+use frame_support::dispatch::DispatchErrorWithPostInfo;
+use frame_system::RawOrigin;
+use sp_core::{offchain::KeyTypeId, sr25519};
+use sp_io::crypto::sr25519_generate;
+use sp_runtime::MultiSignature;
+use sp_std::{num::NonZeroU32, vec::Vec};
+
+use crate::*;
+
+const SEED: u32 = 0;
+const MAX_REVOCATIONS: u32 = 5;
```

Comment:
I suppose we should add a parameter to the delegation config which sets the global max revocations in the future. We wouldn't even have to enforce them, e.g., by throwing if exceeding. Doesn't have to be part of this PR but I still wanted to raise this up to discussion.

PS: I know this code is pretty much what Albi and I wrote before.

Reply:
Added in https://github.com/KILTprotocol/mashnet-node/pull/189/commits/fbb964e3c59d5d67066fd66039854b463a3454f6